distmv/normal.go

```diff
@@ -75,11 +75,17 @@ func NewNormal(mu []float64, sigma mat64.Symmetric, src *rand.Rand) (*Normal, bo
 // ConditionNormal returns {nil, false} if there is a failure during the update.
 // Mathematically this is impossible, but can occur with finite precision arithmetic.
 func (n *Normal) ConditionNormal(observed []int, values []float64, src *rand.Rand) (*Normal, bool) {
+	if len(observed) == 0 {
+		panic("normal: no observed values")
```

Comment:
This is an English thing, so feel free to ignore. The number of zero is singular.

I suspect that in most modern English (certainly here and in American English, but also probably in British Standard English) there are now actually three numbers, singular, plural and (new) nullar which inflects in most cases as the plural. However traditionally, the text would be "no observed value".